.github/workflows/tests.yml

```diff
@@ -117,18 +130,11 @@ jobs:
   mpi_native:
     if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
-    env:
-      HOST_TYPE: ci
-      REDIS_QUEUE_HOST: redis
-      REDIS_STATE_HOST: redis
-    services:
-      redis:
-        image: redis
```

Comment:
This `redis` service wasn't used as everything for the MPI native tests runs through docker-compose (including another Redis instance)